.gitlab-ci.yml

```diff
@@ -519,12 +519,29 @@ filesystem.sh:
   extends: .integration
   parallel:
     matrix:
-    - RUNNER:
-      - aws/fedora-40-x86_64
-    - *rhel_runners
-
+      - CUSTOMIZATION_TYPE: "disk-btrfs"
+        RUNNER:
+          - aws/fedora-40-x86_64
+      - CUSTOMIZATION_TYPE: "filesystem"
+        RUNNER:
+          - aws/rhel-8.10-ga-x86_64
+      - CUSTOMIZATION_TYPE: "disk-plain"
+        RUNNER:
+          - aws/rhel-9.4-ga-x86_64
+      - CUSTOMIZATION_TYPE: "disk-lvm"
+        RUNNER:
+          - aws/rhel-9.6-nightly-x86_64
+      - CUSTOMIZATION_TYPE: "disk-plain"
+        RUNNER:
+          - aws/rhel-10.0-nightly-x86_64
+      - CUSTOMIZATION_TYPE: "disk-lvm"
+        RUNNER:
+          - aws/centos-stream-9-x86_64
+      - CUSTOMIZATION_TYPE: "filesystem"
+        RUNNER:
+          - aws/centos-stream-10-x86_64
```

Comment:
The resulting combination of which test variants are being tested on which OS is weird. Is there any intention behind it? I'd expect to see all variants being run on all runners :thinking:

Reply:
I didn't want the test matrix to grow too much so I added one configuration per distro version.  You're right, it might be a bit confusing.  Also, thinking about it, I think this setup might cause gaps in the future when an old distro version is removed from testing and a new one gets a different configuration.